### PR TITLE
Fix Undeploy operation log and notifications

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #192

### CODE REVIEW

File: src/functions/function-command/undeploy-function.ts [hunk 1]
```diff
@@ -27,17 +26,17 @@ export async function undeployFunction(context: FunctionNode): Promise<string> {
     {
       cancellable: false,
       location: vscode.ProgressLocation.Notification,
-      title: `Undeploy Function: ${context.getName()}.`,
+      title: `Undeploy function ${context.getName()}.`,
```
Comment (lstocchi) [inline src/functions/function-command/undeploy-function.ts:29]: Maybe `Undeploying function {name}`

File: src/functions/function-command/undeploy-function.ts [hunk 2]
```diff
@@ -27,17 +26,17 @@ export async function undeployFunction(context: FunctionNode): Promise<string> {
...
     },
     async () => {
-      const result: CliExitData = await knExecutor.execute(FuncAPI.deleteFunc(context.getName()), process.cwd(), false);
+      const result: CliExitData = await execCmdCli.executeExec(FuncAPI.deleteFunc(context.getName()));
       if (result.error) {
         // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        vscode.window.showErrorMessage(`Fail undeploy Function: ${getStderrString(result.error)}`);
+        vscode.window.showErrorMessage(`Fail undeploy function: ${getStderrString(result.error)}`);
```
Comment (lstocchi) [inline src/functions/function-command/undeploy-function.ts:35]: `Failed to undeploy function {name} - error {err}`

File: src/functions/function-command/undeploy-function.ts [hunk 3]
```diff
@@ -27,17 +26,17 @@ export async function undeployFunction(context: FunctionNode): Promise<string> {
...
         return null;
       }
       // eslint-disable-next-line @typescript-eslint/no-floating-promises
-      vscode.window.showInformationMessage(`Function successfully undeploy Name: ${context.getName()}`);
+      vscode.window.showInformationMessage(`Function ${context.getName()} successfully undeploy`);
```
Comment (lstocchi) [inline src/functions/function-command/undeploy-function.ts:39]: `Function ${context.getName()} successfully undeployed`

**Review by lstocchi — CHANGES_REQUESTED** [review]
cc @mohitsuman as english is not my best and he can suggest better. Thanks

File: src/functions/function-command/undeploy-function.ts (hunk 1 above)
Comment (sudhirverma) [reply src/functions/function-command/undeploy-function.ts]: done

File: src/functions/function-command/undeploy-function.ts (hunk 2 above)
Comment (sudhirverma) [reply src/functions/function-command/undeploy-function.ts]: done

File: src/functions/function-command/undeploy-function.ts (hunk 3 above)
Comment (sudhirverma) [reply src/functions/function-command/undeploy-function.ts]: done

**Review by lstocchi — APPROVED** [review]
LGTM

**Comment by mohitsuman** [comment]
@sudhirverma please fix the merge conflicts and we should be good.

**Pull request merged**